### PR TITLE
refactor(ext/websocket): use specialized ops

### PR DESCRIPTION
**Pull request by bartlomieju**
Instead of relying on `op_ws_send` to send different kinds of messages,
use specialized ops everywhere.

### CODE REVIEW

File: ext/websocket/02_websocketstream.js [hunk 1]
```diff
@@ -265,9 +259,7 @@ class WebSocketStream {
                 }
                 case 3: {
                   /* ping */
-                  await core.opAsync("op_ws_send", this[_rid], {
-                    kind: "pong",
-                  });
+                  await core.opAsync("op_ws_send_pong", this[_rid]);
```
Comment (crowlKats) [inline ext/websocket/02_websocketstream.js:262]: I dont think this pong response is necessary; or at least, we dont send it in the normal websocket impl

Comment (bartlomieju) [reply ext/websocket/02_websocketstream.js:262]: @littledivy suggested that it is necessary

Comment (littledivy) [reply ext/websocket/02_websocketstream.js:262]: this isn't necessary. pong frames are sent automatically but not pong frames (in timeouts)

**Review by mmastrac — APPROVED** [review]
LGTM!

**Review by littledivy — APPROVED** [review]
LGTM

**Pull request merged**